app/src/main/java/com/example/androidchallenge/Data/ApiInterface.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.androidchallenge.Data
+
+
+import com.google.gson.annotations.SerializedName
+import retrofit2.Call
+import retrofit2.http.GET
+
+interface ApiInterface {
```

Comment:
Rename this to `WeatherApi`.

You don't need to call add interface on the name. Think of this "class" as a bridge or a representation of the actual REST API that you are accessing. Having said that, give it a more meaningful name for its propose, since you are getting the weather information, `WeatherApi` seems reasonable.